event-based-simulators/main/scripts/ArgumentsAndCredentialsHandler.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import Environment
+
+from c8y_api import CumulocityApi
+
+
+def argumentsParser():
+    parser = argparse.ArgumentParser(description='Script to export or import profiles data')
+    parser.add_argument('--device-name', '-n', type=str, help='Input device name with double quotes "", i.e "Normal #1"', required=True)
+    parser.add_argument('--action', '-a', choices=['export', 'import'], help='"Export" or "Import" data', required=True)
+    parser.add_argument('--mode', '-m', choices=['all', 'specific']
```

Comment:
- I think it would be better to:
-- dont use "mode" parameter at all; only use "deviceId" parameter:
-- if it is defined, export only for this specific device
-- if it is not defined, try to find all simulators using the existing functionality